backend/apps/volontulo/tests/views/api/organization_contact/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
is there any reason to make this a one-module package instaed of python module?

(See: https://stackoverflow.com/a/7948504)